doc/whats_new/v0.23.rst

```diff
@@ -90,6 +90,9 @@ Changelog
   :user:`Reshama Shaikh <reshamas>`, and
   :user:`Chiara Marmo <cmarmo>`.
 
+- |Fix| Fixed HistGradientBoosting to allow max_depth=1
```

Comment:
```suggestion
- |Fix| Changed the convention for `max_depth` parameter of :class:`ensemlble.HistGradientBoostingClassifier` and :class:`ensemlble.HistGradientBoostingRegressor`. The depth now corresponds to the number of edges to go from the root to the deepest leaf. Stumps (trees with one split) are now allowed.
```

(skip lines where appropriate ;) )

Reply:
Sorry my bad. Would update the changes properly from next time :)